### PR TITLE
[Bexley][WW] Container request email templates

### PR DESCRIPTION
**Pull request by nephila-nacrea**
For https://github.com/mysociety/societyworks/issues/4577.

[skip changelog]

### CODE REVIEW

File: templates/email/bexley/waste/other-reported.html [hunk 1]
```diff
@@ -30,6 +32,16 @@ <h1 style="[% h1_style %]">Your report has been&nbsp;logged</h1>
       <br><br>
       If you need to contact us about this report, please quote your reference number.
     </p>
+  [% ELSIF is_container_request %]
+    <p style="[% p_style %]">
+      Your request has been sent to our waste contractor and will be completed within 10 working days.
```
Comment (dracos) [inline templates/email/bexley/waste/other-reported.html:37]: If you use `[% cobrand.wasteworks_config.request_timeframe || '10 working days' %]` here, as e.g. the default other-reported.html does, we can more easily override the timeframe should they change it (other councils have sometimes, if they're suffering from delays/shortages etc).

Comment (dracos) [reply templates/email/bexley/waste/other-reported.html:37]: This text should presumably also be added to the .txt version as well

File: templates/email/bexley/waste/_sidebar_content.html [hunk 2]
```diff
@@ -4,6 +4,12 @@
   detail = report.detail.remove(address);
   IF is_no_collections_missed_enquiry OR is_missed_collection;
     key = 'Bins reported as missed';
+  ELSIF is_delivery_request;
```
Comment (dracos) [inline templates/email/bexley/waste/_sidebar_content.html:7]: We need to load waste/_email_data.html in the bexley alert-update templates as well

**Review by dracos — CHANGES_REQUESTED** [review]
Looks good, couple of small suggestions is all :)

File: templates/email/bexley/waste/_sidebar_content.html (hunk 2 above)
Comment (nephila-nacrea) [reply templates/email/bexley/waste/_sidebar_content.html:7]: I already do this at the top of `templates/email/bexley/alert-update.html` and `templates/email/bexley/alert-update.txt`, or do you mean somewhere else?

Comment (nephila-nacrea) [reply templates/email/bexley/waste/_sidebar_content.html:7]: Huh, did I forget to push my alerts commit last week?

**Review by dracos — APPROVED** [review]
Thanks :)

**Pull request merged**